src/components/layout.js

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { useStaticQuery, Link, graphql } from "gatsby"
+import styles from "./styles/layout.module.css";
+
+export default function Layout(props) {
+
+    const data = useStaticQuery(
+        graphql
```

Comment:
What's actually happening here (I think) is that graphql is a "tag function", and has a slightly unusual signature — instead of calling it with (), you call it by placing a template string directly after it.

I think a slightly clearer way to write this would be to put the backtick directly after `graphl`, then indent the query inside.